client/src/components/CFooter.style.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+export const Footer = styled.footer`
+  background-color: var(--white);
+  display: flex;
+  flex-direction: column;
+  height: 6rem;
```

Comment:
Adding a small margin in here (maybe 1rem) will help it to be a little bit inset like in the Figma